### PR TITLE
Add docs for new connectivity test buttons added in Octopus 2019.7.13

### PR DESCRIPTION
**Pull request by hnrkndrssn**

### CODE REVIEW

File: docs/deployment-process/issue-tracking/jira.md [hunk 1]
```diff
@@ -140,6 +140,22 @@ When configured, this integration will provide Jira with updates about the progr
 
 ### Testing the Integration
 
+**Octopus 2019.7.13** introduced the ability to verify that a connection between the Octopus server and the Jira cloud/Server instance can be made successfully.
```
Comment (matt-richardson) [inline docs/deployment-process/issue-tracking/jira.md]:
```suggestion
**Octopus 2019.7.13** introduced the ability to verify that a connection can be made successfully between the Octopus server and your Jira Cloud/Server instance.
```

File: docs/deployment-process/issue-tracking/jira.md [hunk 2]
```diff
@@ -140,6 +140,22 @@ When configured, this integration will provide Jira with updates about the progr
...
+
+The **Connect App** `Test` button checks the connectivity for pushing deployment data to a Jira cloud instance.
```
Comment (matt-richardson) [inline docs/deployment-process/issue-tracking/jira.md]:
```suggestion
The **Connect App** `Test` button (found under `Jira Connect App Password`), checks the connectivity for pushing deployment data to your Jira Cloud instance. 
```

File: docs/deployment-process/issue-tracking/jira.md [hunk 3]
```diff
@@ -140,6 +140,22 @@ When configured, this integration will provide Jira with updates about the progr
 
 ### Testing the Integration
 
+**Octopus 2019.7.13** introduced the ability to verify that a connection between the Octopus server and the Jira cloud/Server instance can be made successfully.
+
+The **Connect App** `Test` button checks the connectivity for pushing deployment data to a Jira cloud instance. 
+![Connect App Test button](images/jiraconnectapp_testbutton.png)
+
+:::hint
+For this connectivity test to succeed the Octopus server must be able to connect to https://jiraconnectapp.octopus.com which hosts our Jira plugin and to your Jira cloud instance's Url
```
Comment (matt-richardson) [inline docs/deployment-process/issue-tracking/jira.md]:
```suggestion
For this connectivity test to succeed the Octopus server must be able to connect to both your Jira Cloud instance's url, and to [https://jiraconnectapp.octopus.com](https://jiraconnectapp.octopus.com), which hosts our Jira plugin.
```

File: docs/deployment-process/issue-tracking/jira.md [hunk 4]
```diff
@@ -140,6 +140,22 @@ When configured, this integration will provide Jira with updates about the progr
...
+:::
+
+The **Release Notes** `Test` button checks the connectivity for retrieving work item information from Jira.
```
Comment (matt-richardson) [inline docs/deployment-process/issue-tracking/jira.md]:
```suggestion
The **Release Notes** `Test` button (found under `Jira Password`) checks the connectivity to your Jira Cloud / Server instance for retrieving work item information. 
```

File: docs/deployment-process/issue-tracking/jira.md [hunk 5]
```diff
@@ -140,6 +140,22 @@ When configured, this integration will provide Jira with updates about the progr
...
+
+:::hint
+For this connectivity test to succeed the Octopus server must be able to connect to https://jiraconnectapp.octopus.com which hosts our Jira plugin and to your Jira cloud instance's Url
+:::
+
+The **Release Notes** `Test` button checks the connectivity for retrieving work item information from Jira. 
+![Release Notes Test button](images/jirareleasenotes_testbutton.png)
+
+:::hint
+For this connectivity test to succeed the Octopus server must be able to connect to your Jira cloud/Server instance's Url
```
Comment (matt-richardson) [inline docs/deployment-process/issue-tracking/jira.md]:
```suggestion
For this connectivity test to succeed the Octopus server must be able to connect to your Jira Cloud/Server instance's url.
```

**Comment by hnrkndrssn** [comment]
> Not sure if its Jira Cloud or Jira cloud

Seems the **official** name is actually `Jira Software Cloud` but I'm happy with just `Jira Cloud`

**Pull request merged**